API.pm

```diff
@@ -378,7 +378,7 @@ sub search {
 		limit  => $args->{limit} || DEFAULT_LIMIT
 	};
 
-	if ( $type =~ /album|artist|track|playlist|show_audio|episode_audio/ ) {
+	if ( $type =~ /album|artist|track|playlist|show|episode/ ) {
 		Plugins::Spotty::API::Pipeline->new($self, 'search', sub {
 			my $type = $type . 's';
 			$type =~ s/_audio//;
```

Comment:
Please remove line 384, too. It should no longer be required with your change.

But that earlier assignement of `my $type = $type . 's';` already looks bad... I'll have to review that implementation. Thanks for the heads up!

Reply:
Good point, I've removed line 384 and squashed the commits.